src/run_sign.py

```diff
@@ -8,19 +8,21 @@
 
 import argparse
 import logging
-import os
 import sys
+from pathlib import Path
 
-from manifests.build_manifest import BuildManifest
-from sign_workflow.signer import Signer
+from sign_workflow.sign_artifacts import SignArtifacts
 from system import console
 
+ACCEPTED_SIGNATURE_FILE_TYPES = [".sig", ".asc"]
```

Comment:
Isn't this already `Signer.ACCEPTED_FILE_TYPES`? Use that.

Reply:
No, this is accepted Signature types, different from accepted file types